hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/statemachine/background/BlockDeletingService.java

```diff
@@ -300,7 +313,8 @@ public ContainerBackgroundTaskResult deleteViaSchema1(
         // # of blocks to delete is throttled
         KeyPrefixFilter filter = MetadataKeyFilters.getDeletingKeyFilter();
         List<? extends Table.KeyValue<String, BlockData>> toDeleteBlocks =
-            blockDataTable.getSequentialRangeKVs(null, blockLimitPerTask,
+            blockDataTable.getSequentialRangeKVs(null,
+                (int) blockDeletionLimitPerInterval,
```

Comment:
This should be replaced by numBlocksToDelete for the task.

Reply:
Replaced with blocksToDelete